src/rest_api/classes/protein/widgets/sequences.clj

```diff
@@ -1,21 +1,61 @@
 (ns rest-api.classes.protein.widgets.sequences
   (:require
-    [rest-api.classes.generic-fields :as generic]))
+    [clojure.java.jdbc :as jdbc]
+    [clojure.string :as str]
+    [rest-api.db.sequence :as wb-seq]
+    [rest-api.classes.generic-fields :as generic])
+  (:import (org.biojava.nbio.core.sequence ProteinSequence DNASequence)
+           (org.biojava.nbio.aaproperties PeptideProperties)))
+
+(def abbrev
+  {"A" "Ala"
+   "R" "Arg"
+   "N" "Asn"
+   "D" "Asp"
+   "C" "Cys"
+   "E" "Glu"
+   "Q" "Gln"
+   "G" "Gly"
+   "H" "His"
+   "I" "Ile"
+   "L" "Leu"
+   "K" "Lys"
+   "M" "Met"
+   "F" "Phe"
+   "O" "Pyl*"
+   "P" "Pro"
+   "S" "Ser"
+   "T" "Thr"
+   "W" "Trp"
+   "Y" "Tyr"
+   "V" "Val"
+   "U" "Sec*"
+   "X" "Xaa**"})
```

Comment:
Out of curiosity, why do some amino acids have * in their names?

Reply:
I don't know :)  I got that from the acedb code on the website repo website/lib/WormBase/API/Object/Protein.pm